src/app/core/core.module.ts

```diff
@@ -1,13 +1,24 @@
 import { NgModule } from '@angular/core';
 import { CommonModule } from '@angular/common';
 import { HttpClientModule, HTTP_INTERCEPTORS } from '@angular/common/http';
+
 import { AppInterceptor } from './interceptors/app.interceptor';
+import { SharedModule } from '../shared/shared.module';
+import { StoreModule } from '@ngrx/store';
+import { appReducer } from './store/reducers/app.reducer';
+import { EffectsModule } from '@ngrx/effects';
+import { AppEffects } from './store/effects/app.effects';
+import { StoreDevtoolsModule } from '@ngrx/store-devtools';
 
 @NgModule({
   declarations: [],
   imports: [
     CommonModule,
     HttpClientModule,
+    SharedModule,
```

Comment:
As I understood, you've imported SharedModule to use MatSnackBar?
Better is to extract MatSnackBarModule from material module and import MatSnackBarModule directly in CoreModule because you are not going to use this snackbar in your components as you decided to create special service-wrapper for it
So it should not be a part of material because material is about shared stuff